src/core/ext/transport/binder/wire_format/binder_android.cc

```diff
@@ -16,7 +16,7 @@
 
 #include "src/core/ext/transport/binder/wire_format/binder_android.h"
 
-#if defined(ANDROID) || defined(__ANDROID__)
+#ifdef GPR_SUPPORT_BINDER_TRANSPORT
```

Comment:
same... why not after port_platform.h?

Reply:
Done (was too obligated to include the corresponding header file in .cc)